app/presenters/monitoring-stations/view.presenter.js

```diff
@@ -46,49 +45,47 @@ function go(monitoringStation, auth) {
   return {
     catchmentName,
     enableLicenceMonitoringStationsSetup: FeatureFlagsConfig.enableLicenceMonitoringStationsSetup,
-    enableLicenceMonitoringStationsView: FeatureFlagsConfig.enableLicenceMonitoringStationsView,
     gridReference: gridReference ?? '',
     links: _links(monitoringStationId),
     monitoringStationId,
     pageTitle: _pageTitle(riverName, monitoringStationName),
     permissionToManageLinks: auth.credentials.scope.includes('manage_gauging_station_licence_links'),
     permissionToSendAlerts: auth.credentials.scope.includes('hof_notifications'),
-    restrictionHeading: _restrictionHeading(licenceMonitoringStations),
-    restrictions: _restrictions(licenceMonitoringStations),
+    restrictionHeading: restrictionHeading(licenceMonitoringStations),
+    restrictions: _restrictions(licenceMonitoringStations, FeatureFlagsConfig.enableLicenceMonitoringStationsView),
```

Comment:
I would just have the function check the `FeatureFlagsConfig` directly rather than passing it through as an argument.

That way, when we remove the feature flag we only need to update the method, and not any code that calls it.